### PR TITLE
Fix deadcode and shebangs error

### PR DESCRIPTION
**Pull request by soniasingla**

### CODE REVIEW

**Comment by pawanpraka1** [comment]
@soniasingla can you resolve the conflict and push again?

File: ci/ci-test.sh [hunk 1]
```diff
@@ -1,3 +1,5 @@
+
```
Comment (pawanpraka1) [inline ci/ci-test.sh:1]: please remove this blank line.

**Review by pawanpraka1 — APPROVED** [review]
looks good.

**Comment by pawanpraka1** [comment]
@soniasingla can you DCO sign the last commit and force push?

**Comment by soniasingla** [comment]
Done!

**Review by pawanpraka1 — APPROVED** [review]
looks good.

**Pull request merged**